cluster/examples/kubernetes/ceph/object-bucket-claim-delete.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: ceph-delete-bucket
```

Comment:
As a user, i'm finding it odd to have "delete" in the name of a resource that i don't want deleted until I actually delete it. What about `name: ceph-bucket` with a filename of `object-bucket-claim-delete.yaml`. The filename seems sufficient to represent the difference.

Reply:
The reasoning for the name (valid or not) was that it prevented API resource name collisions for users running through all the OBCs examples consecutively.  OBC names are expected to be set by the user in the wild.  That said, I'd defer to your judgment on it.